lib/awful/widget/watch.lua

```diff
@@ -0,0 +1,84 @@
+---------------------------------------------------------------------------
+--- Watch widget.
```

Comment:
Feel free to add that. Images go to `docs/images` and are linked to... relatively. `docs/03-declarative-layout.md` does `![Example4 screenshot](../images/widgetlayout2.png)`, I guess the same could work for this (as long as ldoc doesn't generate directories...)